recipes/diligent-tools/all/BuildUtils.cmake

```diff
@@ -0,0 +1,399 @@
+if(PLATFORM_WIN32 OR PLATFORM_UNIVERSAL_WINDOWS)
```

Comment:
did you copy it from somewhere? or wrote from scratch?

Reply:
It's from diligent-core
https://github.com/DiligentGraphics/DiligentCore/blob/master/BuildUtils.cmake

Reply:
okay, that's fine. maybe there are ways to export it somehow from the `deligent-core` recipe and use here to avoid copy, but I personally don't know how to do it (maybe @SpaceIm or @prince-chrismc know).

Reply:
I haven't found a way unfortunatelly...
It would be useful, as diligent-tools and diligent-fx recipes are using the same file.